docker-compose.yml

```diff
@@ -1,32 +1,62 @@
 # Note: Requires docker 1.10.0+ and docker-compose 1.6.0+.
-version: "2"
+version: '3.4'
 services:
   db:
     image: postgres:9.4
 
   # run the site with the django dev-server
   web:
```

Comment:
Maybe this container should have a different name than `web`. Maybe "staticweb" or "assets" or "jeff"?

Reply:
I left it `web` just because I'm so used to doing `docker-compose up web` and the django app is a dependency of `web` and so they both start and it's all good. I'm happy to name it whatever though since mostly make takes care of everything.